articles/understanding-systemd/index.md

```diff
@@ -0,0 +1,248 @@
+## Introduction
+From the official website, systemd (intentionally spelt with a small 's') is described as a suite of tools that forms the basic building block for a Linux system.
```

Comment:
@Bashorun97, sentences are long, and we also prefer American English for example, "spelt" should be "spelled". Run the article on Grammarly and Hemingway Editor to fix grammar issues.
Ping me when you're done.

Reply:
consider replacing 'spelt' with 'spelled'.